From 8b369b833e3168506342dfa3a237f28368dce98d Mon Sep 17 00:00:00 2001
From: Hans Rosenfeld <hans.rosenfeld@joyent.com>
Date: Thu, 20 Dec 2018 10:09:53 +0000
Subject: [PATCH] OS-7404 mdb: add KREG_FSBASE Reviewed by: John Levon
 <john.levon@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: John Levon <john.levon@joyent.com>

---
 usr/src/cmd/mdb/intel/mdb/mdb_kreg.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/usr/src/cmd/mdb/intel/mdb/mdb_kreg.h b/usr/src/cmd/mdb/intel/mdb/mdb_kreg.h
index a3edf864d7..08d65c0cef 100644
--- a/usr/src/cmd/mdb/intel/mdb/mdb_kreg.h
+++ b/usr/src/cmd/mdb/intel/mdb/mdb_kreg.h
@@ -75,6 +75,7 @@ typedef uint32_t kreg_t;
 #define	KREG_ES		KDIREG_ES
 #define	KREG_FS		KDIREG_FS
 #define	KREG_GS		KDIREG_GS
+#define	KREG_FSBASE	KDIREG_FSBASE
 #define	KREG_GSBASE	KDIREG_GSBASE
 #define	KREG_KGSBASE	KDIREG_KGSBASE
 #define	KREG_TRAPNO	KDIREG_TRAPNO
-- 
2.21.0

